sdk/eas/README.md

```diff
@@ -0,0 +1,57 @@
+# @settlemint/sdk-eas
+
+Ethereum Attestation Service (EAS) integration for SettleMint SDK.
+
+## Features
+
+- [ ] Attestation submit/parse
+- [ ] Graph query helpers
+- [ ] Schema creation utils
+
+## Installation
+
+```bash
+npm install @settlemint/sdk-eas
+```
+
+## Usage
```

Comment:
Put "Usage and Features" in the `docs/ABOUT.md` file. The main readme of every package is updated automatically using a script, so this will be overwritten. You can test doc generation by running "bun docs" on the root of this repo.